assets/app/features/upgrade/templates/landing-page_step3.partial.jade

```diff
@@ -0,0 +1,12 @@
+.upgrade(ng-class="{'active': upgradeLandingVm.mode.valid}")
+    .step-title.clearfix
+        crowbar-circle.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid}") 3
+        h2.h4(translate='') upgrade.steps.landing.upgrade.title
+    p(translate='') upgrade.steps.landing.upgrade.description
```

Comment:
Alignment is yet to be implemented.
For further reference, please refer to https://github.com/crowbar/crowbar-ui/blob/wiki-assets/upgrade67/crowbar-wizard-comp-colours-and-spacing.pdf